main.go

```diff
@@ -427,7 +435,7 @@ func generateStaticMetadata(md []byte) (b []byte, metadata map[string]string) {
 func generateStyledCodeHTML(html []byte) []byte {
 	doc, err := goquery.NewDocumentFromReader(bytes.NewReader(html))
 	if err != nil {
-		log.Event(nil, "Failed to read html file", log.Error(err))
+		log.Error(context.TODO(), "Failed to read html file", err)
```

Comment:
```suggestion
		log.Error(context.TODO(), "failed to read html file", err)
```
you stopped lower-casing error messages a while back